### ISSUE TITLE
Angular 6 PWA wont loads in OS X and iOS browsers

### ISSUE THREAD

**Issue by nategurian** [angular/angular-cli#12699]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [x ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions
OS: Mac High Sierra
iOS: 11.3
Node: 9.2.0
Angular: 6.0.8

### Repro steps
navigate to https://supportchatdev.herokuapp.com on any iOS or MAC Browser

### The log given by the failure

No Visible errors in console

### Desired functionality
I would like to run ng serve on my MAC and test my Angular PWA.

### Mention any other details that might be useful
I deployed an Angular app to Heroku and when testing the site above in the repro steps I was able to access the app just fine on any Windows browser except IE which I know I dont have the polyfills for.

When I went to that site on my iPHONE or on my MAC, the site would not load . I do not get any error other than a generic timeout from the browser (as if I was not connected to the internet). I cloned the same repo I use on my windows machine onto my mac and when I run ng serve the app builds and compiles no problem, but when I navigate to a browser (Safari, Chrome, Firefox) I just get a generic browser timeout and nothing useful in the console. If I turn WIFI off, I can access the site (localhost:4200), but the moment I turn wifi back on i get to the same timeout

I am not sure which files would be useful for people to see, so I can show those upon request.
I know the error information sucks (as I dont get any error from the console) but just wondering if there might be a solution to the problem.

Thanks for any help or insight

**Comment by alan-agius4** [comment]
This seems like a bug but we'll need to look at a reproduction to find and fix the problem. Can you setup a minimal repro please?

You can read [here](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue) why this is needed. A good way to make a minimal repro is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Then you can push this repository to github and link it here.

This might be related to your directory structure so its really important to get an accurate repro to diagnose this.

**Comment by cezaraf** [comment]
was solved? how??

**Comment by nategurian** [comment]
@alan-agius4 I am not able to recreate the issue. When I download the initial repository from https://github.com/luixaviles/socket-io-typescript-chat I am able to run it no problem. It is not until I added my own packages that it stopped working. I will need to figure this one out some other way. Thanks

**Comment by nategurian** [comment]
> was solved? how?? i have the same problem...

@cezaraf 
I have not solved it yet. Still trying

**Comment by alan-agius4** [comment]
I'm sorry, but we can't reproduce the problem following the instructions you provided.
Remember that we have a large number of issues to resolve, and have only a limited amount of time to reproduce your issue.
Short, explicit instructions make it much more likely we'll be able to reproduce the problem so we can fix it.

If the problem persists, please open a new issue following [our submission guidelines](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-submitting-an-issue).

A good way to make a minimal repro is to create a new app via `ng new repro-app` and adding the minimum possible code to show the problem. Then you can push this repository to github and link it here.
